app/assets/javascripts/cascade/level/rotator.js

```diff
@@ -31,7 +31,7 @@ $(function () {
         var beforeSlideChange = function(slider) {
             if ( !isSliderMasthead ) { return; }
 
-            $header.fadeOut();
```

Comment:
By jump, I mean the block gets larger. I don't know what else it could do not being fixed height. What did you mean originally by jump?